qbft/spectest/tests/proposal/prepared_previously_no_prepare_justification_quorum.go

```diff
@@ -17,9 +17,12 @@ func PreparedPreviouslyNoPrepareJustificationQuorum() *tests.MsgProcessingSpecTe
 		testingutils.TestingPrepareMessage(ks.Shares[2], types.OperatorID(2)),
 	}
 	rcMsgs := []*qbft.SignedMessage{
-		testingutils.TestingRoundChangeMessageWithRound(ks.Shares[1], types.OperatorID(1), 2),
-		testingutils.TestingRoundChangeMessageWithRound(ks.Shares[2], types.OperatorID(2), 2),
-		testingutils.TestingRoundChangeMessageWithRound(ks.Shares[3], types.OperatorID(3), 2),
+		testingutils.TestingRoundChangeMessageWithRoundAndRC(ks.Shares[1], types.OperatorID(1), 2,
```

Comment:
why this change?

Reply:
due to my mistake in a previous PR (https://github.com/bloxapp/ssv-spec/pull/162/files#r1120058796)